monkey/agent_plugins/runner/plugin_runner.py

```diff
@@ -0,0 +1,42 @@
+from io import BytesIO
+from ipaddress import IPv4Address
+from multiprocessing import Event
+from typing import Any, Dict
+from unittest.mock import MagicMock
+from uuid import UUID
+
+from agent_plugins.exploiters.hadoop.src.plugin import Plugin
+from tests.monkey_island import InMemoryFileRepository
+
+from common import OperatingSystem
+from common.event_queue import IAgentEventPublisher
+from infection_monkey.model import TargetHost
+from monkey_island.cc.repositories import AgentBinaryRepository
```

Comment:
This is the wrong `AgentBinaryRepository`. There's a separate one in `infection_monkey`.